unittests/runtime/Mutex.cpp

```diff
@@ -856,11 +856,12 @@ template <typename RW> void readWriteLockCacheExampleThreaded(RW &lock) {
             if (trace)
               printf("Worker[%d] miss for key = %d.\n", i, key);
             found = false; // cache miss, need to grab write lock
+          } else {
+            if (trace)
+              printf("Worker[%d] HIT for key = %d, value = %d.\n", i, key,
+                     value->second);
+            found = true; // cache hit, no need to grab write lock
           }
-          if (trace)
-            printf("Worker[%d] HIT for key = %d, value = %d.\n", i, key,
-                   value->second);
-          found = true; // cache hit, no need to grab write lock
```

Comment:
I also added the missing `else` in this code, but I'm not sure if it is okay to fix it in this PR as well?

Reply:
I would put it in a different PR. Different high level thing.